<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>eventRegistration</class>
 <widget class="QMainWindow" name="eventRegistration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>eventRegistration</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>110</y>
      <width>171</width>
      <height>191</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="firstNameLabel">
         <property name="text">
          <string>First Name:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="lastNameLabel">
         <property name="text">
          <string>Last Name: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="emailAddressLabel">
         <property name="text">
          <string>Email Address: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="phoneNumberLabel">
         <property name="text">
          <string>Phone Number: </string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="ageLabel">
         <property name="text">
          <string>Age: </string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="0" column="1">
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLineEdit" name="firstNameLineEdit">
         <property name="inputMask">
          <string>aaaaaaaaaaaaaa</string>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lastNameLineEdit">
         <property name="inputMask">
          <string>aaaaaaaaaaaaaa</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="emailAddressLineEdit">
         <property name="inputMask">
          <string>nnnnnnnnnnnnnnnnnnnnxnnnnnnnnnnnnnnnnnxnnnn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="phoneNumberLineEdit">
         <property name="inputMask">
          <string>999-999-9999</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ageLineEdit">
         <property name="inputMask">
          <string>000</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="1" column="0">
      <widget class="QPushButton" name="submitButton">
       <property name="text">
        <string>Submit</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="saveAllButton">
       <property name="text">
        <string>Save All</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QPushButton" name="cancelButton">
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLabel" name="successLabel">
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
